arch/xtensa/src/esp32/Make.defs

```diff
@@ -39,8 +39,8 @@ CMN_CSRCS += xtensa_modifyreg8.c xtensa_modifyreg16.c xtensa_modifyreg32.c
 CMN_CSRCS += xtensa_puts.c xtensa_releasepending.c xtensa_releasestack.c
 CMN_CSRCS += xtensa_reprioritizertr.c xtensa_schedsigaction.c
 CMN_CSRCS += xtensa_sigdeliver.c xtensa_stackframe.c xtensa_udelay.c
-CMN_CSRCS += xtensa_unblocktask.c xtensa_usestack.c
-CMN_CSRCS += esp32_systemreset.c esp32_resetcause.c
+CMN_CSRCS += xtensa_unblocktask.c xtensa_usestack.c xtensa_swint.c
+CMN_CSRCS += esp32_systemreset.c esp32_resetcause.c xtensa_switchcontext.c
```

Comment:
```suggestion
CMN_CSRCS += xtensa_switchcontext.c
```
Unrelated to this PR, but `esp32_systemreset.c` and `esp32_resetcause.c` are not common Xtensa sources.
Could you please move them to the `CHIP_CSRCS` variable?